static/app/stores/customMeasurementsStore.tsx

```diff
@@ -0,0 +1,57 @@
+import {createStore} from 'reflux';
```

Comment:
I think we're moving towards using react contexts instead of reflux stores if possible, so should we re-write this to use contexts instead?

Reply:
😄 good point. I rewrote it using contexts!